README.md

```diff
@@ -1,2 +1,9 @@
-# sonar-scanner-cli-docker
-Docker image for SonarScanner CLI
+# Sonar Scanner CLI 
+
+This is the Git repository that contains source for [SonarScanner CLI](http://www.onet.pl) Docker images. Once approved, you will be able to pull the images and a see the full readme on DockerHub.  
```

Comment:
target of the `SonarScanner CLI` link is... unexpected

Reply:
oops, forgot to push the update 😄